### PR TITLE
fix generated code for emtpy value of @DefaultStringSet

### PR DESCRIPTION
**Pull request by dodgex**
see https://github.com/excilys/androidannotations/pull/1555#issuecomment-154579420

### CODE REVIEW

**Comment by dodgex** [comment]
@WonderCsabo maybe extract the `keyRes` param into a separate annotation? to allow providing a resource for the key without requiring a default value?

File: AndroidAnnotations/androidannotations-core/androidannotations-api/src/main/java/org/androidannotations/annotations/sharedpreferences/DefaultStringSet.java [hunk 1]
```diff
@@ -24,8 +24,13 @@
 
 /**
  * <p>
- * Use on methods in {@link SharedPref} annotated class. The generated method
- * will return an empty {@link java.util.Set} of Strings by default.
+ * Use on methods in {@link SharedPref} annotated class to specified the default
+ * value of this preference.
+ * </p>
+ * <p>
+ * The annotation value must be an <code>String[]</code>. The generated method
```
Comment (WonderCsabo) [inline AndroidAnnotations/androidannotations-core/androidannotations-api/src/main/java/org/androidannotations/annotations/sharedpreferences/DefaultStringSet.java]: a String[]

Comment (dodgex) [inline AndroidAnnotations/androidannotations-core/androidannotations-api/src/main/java/org/androidannotations/annotations/sharedpreferences/DefaultStringSet.java]: whoops, i copied this from `@DefaultInt` and didnt recognized the n.

**Comment by WonderCsabo** [comment]
Thanks for this!

**Pull request merged**